schemas/questionnaire_v1.json

```diff
@@ -141,48 +113,71 @@
             "$ref": "common_definitions.json#/id"
           },
           "title": {
-            "type": "string"
-          },
-          "hide_in_navigation": {
-            "type": "boolean"
+            "type": "string",
+            "description": "Title for a given id."
```

Comment:
This description isn't very descriptive.
Is this the title that will be displayed in the navigation?